src/iidm/NodeBreakerVoltageLevel.cpp

```diff
@@ -336,13 +365,6 @@ void NodeBreakerVoltageLevel::removeSwitch(const std::string& switchId) {
     getNetwork().remove(aSwitch.get());
 }
 
-void NodeBreakerVoltageLevel::setNodeCount(unsigned long nodeCount) {
-    unsigned long oldCount = m_graph.getVertexCount();
-    for (unsigned long i = oldCount; i < nodeCount; ++i) {
-        m_graph.addVertex();
-    }
-}
```

Comment:
Some changes are missing: the `attach` method has not been updated

Reply:
Yes it does : https://github.com/powsybl/powsybl-iidm4cpp/pull/137/files#r451423417